Telerik.JustMock/Helpers/DynamicTypeHelper.cs

```diff
@@ -0,0 +1,66 @@
+﻿/*
+ JustMock Lite
+ Copyright © 2010-2018 Telerik EAD
```

Comment:
Could change the copyright notice as it should be 2018 only. This is applicable to all new files.

Reply:
@mihail-vladov fixed with [41d482f](https://github.com/telerik/JustMockLite/pull/42/commits/41d482f390d2ad05ddea272d25c4b14dca4e999b)